logisland-framework/logisland-scripting/logisland-scripting-base/src/main/java/com/hurence/logisland/jsr223/BindingsImpl.java

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright (C) 2016 Hurence (bailet.thomas@gmail.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hurence.logisland.jsr223;
+
+import java.util.Collection;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Set;
+
+import javax.script.Bindings;
+
+public class BindingsImpl implements Bindings {
+	private Map<String, Object> map;
+	
+	public BindingsImpl(Map<String, Object> map) {
+		Objects.requireNonNull(map);
+		
+		this.map = map;
```

Comment:
It would be safer, but less efficient. As this is only going to be used by LogIsland internally, we could leave it as-is for better performances.